timer.js

```diff
@@ -1,11 +1,16 @@
 const totalTime = 30;
-let timeLeft = totalTime;
+let timeLeft;
 
 function timer() {
     setTimeout(() => {
         if (timeLeft > 0) {
             timer();
         } else {
+            let playerScore = document.querySelector('#player_score').textContent.replace('Score: ', '');
+
+            document.querySelector('#game_over').style.display = 'flex';
+            document.querySelector('#game').style.display = 'none';
```

Comment:
there is a lot of this kind of line in the application now. Can we create a function called something like: `hideScreen` that takes the screen as a param and sets it to `none`, make the code more DRY